tests/phpunit/tests/rest-api/rest-search-controller.php

```diff
@@ -79,6 +86,14 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 			)
 		);
 
+		self::$my_primed_post_ids = $factory->post->create_many(
```

Comment:
Do we need to create new fixtures here? Why can't we reuse the existing title ones for instance.

Reply:
ei @TimothyBJacobs I've created them for clarity and readability, but this is a personal preference.

Reply:
refactored to use previous fixture

Reply:
Thanks @manuelRod. Consolidating fixtures is important to keep our test suite performing well too.